kie-server-parent/kie-server-wars/kie-server/src/main/webc-resources/README.md

```diff
@@ -38,38 +42,29 @@ This instruction describes all steps to install KIE Server on Tomcat 9 standalon
     As a next step, configure a pooling data source, that relies on XA data source for creating new connections. 
     In the example, this data source is named “poolingXaDs”.
     
-    Edit TOMCAT_HOME/conf/context.xml and add following within Context tags of the file:
-    ``` 
-    <Resource 
+    Edit TOMCAT_HOME/conf/server.xml and add following within GlobalNamingResources tags of the file (e.g. after the UserDatabase Resource definition):
+    ```
+	<Resource 
         auth="Container" 
-        databaseName="${datasource.dbName}" 
         description="XA Data Source" 
-        factory="org.apache.tomcat.jdbc.naming.GenericNamingResourcesFactory" loginTimeout="0" 
+        factory="org.apache.tomcat.jdbc.naming.GenericNamingResourcesFactory"
+		loginTimeout="0" 
         name="xads"
         uniqueName="xads" 
+        testOnBorrow="false" 
+        driverType="4"
+
         portNumber="${datasource.port}"
         serverName="${datasource.hostname}" 
-        testOnBorrow="false" 
```

Comment:
Why is this removed? Did it cause any issues ?

Reply:
it's not removed, just the order of attributes has been changed.

Reply:
@SimonDallaway was the order of attributes changed to comply with some commonly used convention?

Reply:
Hi,

I think I was just trying to ensure that the properties with literal values were grouped first followed by the ones that were variables.

My major concern was getting the instructions so that they worked and were presented logically. I'm a definite newbie to all of this.